platform/src/main/java/org/stellar/anchor/platform/component/sep/SepBeans.java

```diff
@@ -212,9 +212,15 @@ Sep38Service sep38Service(
       Sep38Config sep38Config,
       AssetService assetService,
       RateIntegration rateIntegration,
+      CustomerIntegration customerIntegration,
       Sep38QuoteStore sep38QuoteStore,
       EventService eventService) {
     return new Sep38Service(
-        sep38Config, assetService, rateIntegration, sep38QuoteStore, eventService);
+        sep38Config,
+        assetService,
+        rateIntegration,
+        customerIntegration,
+        sep38QuoteStore,
+        eventService);
```

Comment:
We don't need `customerIntegration` here anymore, right?

Reply:
That's right. Changed.